File: Arrays 101/1. Max Consecutive Ones/find_max.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        counter = 0
        max_1 = 0
        for item in nums:
            if item == 1:
                counter += 1
                max_1 = max(max_1, counter)
            else:
                counter = 0
        return max_1

File: 1. Arrays 101/485. Max Consecutive Ones/find_max.cpp
class Solution
{
public:
    int findMaxConsecutiveOnes(vector<int> &nums)
    {
        int counter = 0;
        int max = 0;
        for (int i = 0; i < nums.size(); i++)
        {
            if (nums[i] == 1)
            {
                counter++;
                max = max > counter ? max : counter;
            }
            else
            {
                counter = 0;
            }
        }
        return max;
    }
};